src/util.h

```diff
@@ -355,4 +355,14 @@ std::string CopyrightHolders(const std::string& strPrefix);
  */
 int ScheduleBatchPriority(void);
 
+//! Simplification of std insertion
+template <typename Tdst, typename Tsrc>
+void insert(Tdst& dst, const Tsrc& src) {
```

Comment:
should we call these `insert`? I think it's a bit confusing for readability, as it implies the built-in std function is used in ways that the original one cannot be.

Reply:
I was considering putting them under a bitcoin:: namespace or somehing to emphasize they're actually not `std` but since we are no longer `using namespace std` it felt unnecessary. I'm up for renaming if it still feels confusing though.

Reply:
I ended up moving into `util::` after all.